logging/bin/getlogs.py

```diff
@@ -206,6 +230,8 @@ def get_arguments():
 
 args = get_arguments() ##Creates "args" dictionary that contains all user submitted options. Print "args" to debug values. Note that the 'dest' value for each argument in argparser object is its key.
 validate_input(args)
+if args['portforward']:
+    open_port(args)
```

Comment:
Should we just pass `args['portforward']`? Similar to my comment above, it can get dangerous just passing dictionaries around. The minimal amount of arguments necessary should be passed.